torch/_ops.py

```diff
@@ -430,6 +538,7 @@ def mode_stack_per_key():
 def push_mode_for_key(key, mode):
     assert isinstance(key, torch._C.DispatchKey)
     assert isinstance(mode, torch.utils._python_dispatch.TorchDispatchMode)
+    assert key != torch._C.DispatchKey.PreDispatch
```

Comment:
Is this function actually used anywhere if we're no longer using it for PreDispatch?